src/Volume.ts

```diff
@@ -248,9 +270,8 @@ export default class Volume {
     if (this.channels.every((element) => element.loaded)) {
       this.loaded = true;
     }
-    for (let i = 0; i < this.volumeDataObservers.length; ++i) {
-      this.volumeDataObservers[i].onVolumeData(this, batch);
-    }
+    batch.forEach((channelIndex) => this.channelLoadCallback?.(this.loadSpec.url, this, channelIndex));
```

Comment:
probably should document somewhere (in IVolumeLoader) the order in which the two possible channel loaded callbacks will be called.